pkg/operator/k8sutil/pvc.go

```diff
@@ -31,10 +33,24 @@ func ExpandPVCIfRequired(client client.Client, desiredPVC *v1.PersistentVolumeCl
 		logger.Debugf("desired or current size are not specified for PVC %q", currentPVC.Name)
 		return
 	}
+
 	if desiredSize.Value() > currentSize.Value() {
+		// get StorageClass
+		storageClass := &storagev1.StorageClass{}
+		err := client.Get(context.TODO(), types.NamespacedName{Name: *(currentPVC.Spec.StorageClassName)}, storageClass)
+		if err != nil {
+			logger.Errorf("failed to get storageClass. %v", err)
```

Comment:
Put the name of the SC in the log message.

Reply:
Done